### ISSUE TITLE
Bug: After importing a CSV resource:clean fails

### ISSUE THREAD

**Issue by paavo** [neos/redirecthandler-ui#96]
After importing a CSV file (urls_to_import.csv) with URLs in redirecthandler-ui i get an error when publishing resources (with resource:clean). 

> _Error: Could not publish resource urls_to_import.csv with SHA1 hash xxx of collection persistent because there seems to be no corresponding data in the storage._

I guess the csv Files get deleted but not removed from persistentresource Tables @Sebobo?

resource:clean helps to fix the problem.

**Comment by Sebobo** [comment]
Do you get `Could not delete csv file after importing redirects` in your logs?

**Comment by paavo** [comment]
I get the error when publishing ressources, after i imported a redirects file @Sebobo

**Comment by Sebobo** [comment]
Yes but do you get the mentioned error in your logs directly after you import the csv, meaning the current cleanup fails?

**Comment by paavo** [comment]
> do you get the mentioned error in your logs directly after you import the csv, meaning the current cleanup fails

I don't think so @Sebobo 
I only noticed the issue a few times when MagicWand failed publishing (after cloning remote to local dev environment)

**Comment by Sebobo** [comment]
@paavo  I cannot reproduce this problem locally. The file is correctly removed after import.

Can you do another import with a small file that only contains one redirect and check the log output related to that import, if there is any warning.
Do you have any special storage, or the default filesystem one?
